RULES.md

```diff
@@ -97,6 +97,7 @@ Additional details regarding the notices' context is provided in [`NOTICES.md`](
 | [`LocationWithUnexpectedStopTimeNotice`](#LocationWithUnexpectedStopTimeNotice)                                 | A location in `stops.txt` that is not a stop is referenced by some `stop_times.stop_id`.                                                               |
 | [`MissingCalendarAndCalendarDateFilesNotice`](#MissingCalendarAndCalendarDateFilesNotice)                       | Missing GTFS files `calendar.txt` and `calendar_dates.txt`.                                                                                            |
 | [`MissingLevelIdNotice`](#MissingLevelIdNotice)       	                                                      | `stops.level_id` is conditionally required.                                                                                                            |
+| [`MissingRecommendedFileNotice`](#MissingRecommendedFileNotice)                                                       | A recommended file is missing.                                                                                                                            |
```

Comment:
We are missing the part this link brings the user to. It is usually a longer description and it appears lower in the document. It could be:

> #### MissingRequiredFileNotice
> The GTFS Best Practice recommends a file be included, but it's missing.
> 
> #### References:
> [GTFS Best Practices](https://gtfs.org/schedule/best-practices/)

Reply:
⚠️ @KClough We are deprecating NOTICES.md in PR #1132